docs/notebooks/index.md

```diff
@@ -51,9 +51,10 @@ The Digital Assets contract family implements a basic digital asset for bitmap i
 ### Inference
 
 
-The Inference contract family provides contracts for creating a confidentiality preserving policy-wrapper around the usage of a machine learning (ML) model. At its core, the implementation uses a token contract to  specify and enforce policies to be followed while using the ML model for inferecing operations. 
+The Inference contract family provides contracts for creating a confidentiality
+preserving policy-wrapper around the usage of a machine learning (ML) model.
+At its core, the implementation uses a token contract to  specify and enforce
```

Comment:
We shouldn't assume the reader knows what a "token contract" is. It would be helpful to add a link to "token contract" to documentation where the reader can find more information on the underlying implementation.

Reply:
thanks, please see comment above for fix.